source/components/molecules/EditableList/EditableList.js

```diff
@@ -20,6 +21,8 @@ const EditableListItem = styled.View`
   background-color: transparent;
   border-radius: 4.5px;
   margin-bottom: 10px;
+  ${({ theme, error }) =>
+    error?.isValid || !error ? '' : `border: solid 1px ${theme.colors.primary.red[0]}`}
```

Comment:
I think this could be written in this format instead. I think it's valid to use with styled-components.
error?.isValid || error && `border: solid 1px ${theme.colors.primary.red[0]}`

Nothing wrong to use the ternary operator. But the return of an empty string could be avoided :)

Reply:
Fixed